src/main/java/network/brightspots/rcv/TieBreak.java

```diff
@@ -88,6 +91,10 @@
     this.candidatePermutation = candidatePermutation;
   }
 
+  static void setRandomSeed(int randomSeed) {
+    random = new Random(randomSeed);
```

Comment:
Minor code smell for me here... I think it would be better if you actually had a `randomSeed` field, not just a `random` field, stored in the class, and here you both set the `randomSeed` field and initialize `random` using that field. From a cursory Google search, it seems to be relatively unintuitive to extract a seed from an initialized `Random` object. I think we probably want both fields to be `final` too.

Reply:
We're not accessing `randomSeed` in `TieBreak`, though. If we decide we want to in the future, we can just refactor it accordingly.

And I can't make `random` final because it doesn't necessarily get initialized.